google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractResultSet.java

```diff
@@ -546,6 +559,7 @@ static Object decodeArrayValue(Type elementType, ListValue listValue) {
               listValue.getValuesList(),
               input -> input.getKindCase() == KindCase.NULL_VALUE ? null : input.getBoolValue());
         case INT64:
+        case ENUM:
           // For int64/float64 types, use custom containers.  These avoid wrapper object
```

Comment:
misplaced comment

Reply:
Yes, will refactor it once https://github.com/googleapis/java-spanner/pull/2143 is merged.